packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# Unreleased
+# Unreleased (1.8.0)
```

Comment:
Don't you mean to put this down on a separate line?

Reply:
This is my way of suggesting the next version number. It's still unreleased.